tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -118,8 +118,8 @@ Status StorageManagerCanonical::init() {
   tile_cache_ =
       tdb_unique_ptr<BufferLRUCache>(tdb_new(BufferLRUCache, tile_cache_size));
 
-  // GlobalState must be initialized before `vfs->init` because S3::init calls
-  // GetGlobalState
+  // GlobalState must be initialized before initializing the VFS because
+  // S3::init calls GetGlobalState
```

Comment:
You can just delete this comment instead of rewording it. Its not longer true that `S3::init` calls `GetGlobalState`, I just forgot to delete this when I did that PR.